pkg/util/ip/addr.go

```diff
@@ -43,3 +43,21 @@ func lastIP(subnet net.IPNet) net.IP {
 	}
 	return end
 }
+
+// lastUsableIP returns second to last IP of a subnet
+func LastUsableIP(subnet net.IPNet) net.IP {
+	// FIXME(dulek): This should have proper IPv6 support, but for now whatever, Kuryr doesn't support it yet.
+	ip := lastIP(subnet)
+	// And this second to last (last usable)
+	ip[len(subnet.IP)-1] &= 0xFE
+	return ip
+}
+
+// FirstUsableIP returns second  IP of a subnet
+func FirstUsableIP(subnet net.IPNet) net.IP {
+	// FIXME(dulek): This should have proper IPv6 support, but for now whatever, Kuryr doesn't support it yet.
+	ip := subnet.IP
+	// And this second one (first usable)
+	ip[len(ip)-1] |= 0x01
```

Comment:
as above, the base IP is guaranteed to be an even number, so this is really `+ 1`